Controllers/HomeController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("", Name ="Home", Order = 0)]
+		[Route("app", Name = "app", Order = 0)]
+		public IActionResult App()

```

Comment:
Following routes has been added on single action now. 

		[Route("")] // Default localhost
		[Route("app")] // to handle localhost/app
		[Route("app/{path}")] // to handle localhost/app/{anything}
		[Route("app/product/{productId}")] // to handle product pages